src/Control/Monad/Metrics.hs

```diff
@@ -203,6 +205,20 @@ gauge' =
 gauge :: (MonadIO m, MonadMetrics m) => Text -> Int -> m ()
 gauge = gauge'
 
+-- | See 'System.Metrics.Distribution.Gauge.dec'.
+--
+-- * /Since v0.2.2.0/
+gaugeDec :: (MonadIO m, MonadMetrics m) => Text -> m ()
+gaugeDec name =
```

Comment:
```suggestion
gaugeDecrement name =
```

I think I'd prefer to see the longer name, since that's how they are for eg `counter`s.